### DISCUSSION TITLE
[Issue] health config file - empty values in cached file

### DISCUSSION THREAD

**Discussion by LocalHeroPro** [spatie/laravel-health#145] [General]
Steps to reproduce issue:
1. Fill up `config/health.php` file:
```php
//
'mail' => [
    'to' => config('mail.admins'),

    'from' => [
        'address' => env('MAIL_FROM_ADDRESS', 'hello@example.com'),
        'name' => env('MAIL_FROM_NAME', 'Example'),
    ],
],
//
```
2. Make config cache:
```bash
 $ php artisan config:cache
```
3. In cached file `bootstrap/cache/config.php` we have:
```php
// health config
'mail' => 
array (
  'to' => NULL, // wrong value
  'from' => 
  array (
    'address' => 'noreply@example.com', // value from `.env` file
    'name' => 'Awesome App', // value from `.env` file
  ),
),
// mail config
'admins' => 
array (
  'emails' => 
  array (
    0 => 'admin@example.com', // good value
  ),
),
//
```
4. So when we using `env()` values are get in proper way. When we pointing to other config file using `config()` values are not proper.
5. Same scenario I used in other config files, so in `config/mail.php` and `config()` works as expected.

**Comment by freekmurze** [comment]
This is not an issue that can be solved by this package. Open an issue at the Laravel framework to get some feedback on this.
